src/pages/billing/List.js

```diff
@@ -251,14 +263,22 @@ export const BillingList = _.flow(
           }), billingProjects)
         ]),
         creatingBillingProject && h(NewBillingProjectModal, {
+          billingAccounts,
           onDismiss: () => this.setState({ creatingBillingProject: false }),
           onSuccess: () => {
             this.setState({ creatingBillingProject: false })
             this.loadProjects()
           }
         }),
-        !!selectedName && billingProjects && h(ProjectDetail, { key: selectedName, project: _.find({ projectName: selectedName }, billingProjects) }),
-        isBusy && spinnerOverlay
+        !!selectedName && billingProjects && h(ProjectDetail, {
+          key: selectedName,
+          project: _.find({ projectName: selectedName }, billingProjects),
+          billingAccounts,
+          hasBillingScope: Auth.hasBillingScope(),
```

Comment:
This prop seems unnecessary. `ProjectDetail` should be able to use `Auth.hasBillingScope()` directly.

Reply:
The reason I am using this as a prop here is because the detail page does not know when the scope has changed. The scope changes are handled in List.js (at the page level). I could push this down to the Project Detail component, but the project detail would then need to track these changes in componentDidUpdate which can get messy as it would need to track the previous state of the billing scope and possibly some other additional state.